static/src/javascripts/projects/common/modules/discussion/comment-count.js

```diff
@@ -31,9 +31,9 @@ define([
         defaultTemplate = commentCountTemplate;
 
     function getContentIds() {
-        return _.map(qwery('[' + attributeName + ']'), function (el) {
+        return _.uniq(_.map(qwery('[' + attributeName + ']'), function (el) {
```

Comment:
I was about to ask where the hell `.uniq(...)` came from when I noticed it was not Scala... Think I am tired :sleeping: